azure-pipelines.yml

```diff
@@ -383,7 +383,14 @@ jobs:
       Contents: '**\*'
       TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
+  - task: NuGetCommand@2

```

Comment:
I think building nuget should be in a separate release pipeline where we actually publish it, not in the build pipeline